### ISSUE TITLE
AuthorOrdering schema requires an id

### ISSUE THREAD

**Issue by stlozhaw** [sustainability-zhaw/extractor-project#2]
- Add mutation with upsert: Without id will always add to the existing list (Upsert has no way to remove entries like update)
- Update mutation:  Unable to remove existing entries without id

Update mutation will throw the following error:
`couldn't rewrite mutation updateInfoObject because failed to rewrite mutation payload because id is not provided`

Maybe AuthorOrdering should have the author (fullname) as id.
AuthorOrdering should be rebuild on every update based on the current author list.

**Comment by stlozhaw** [comment]
@phish108 Do we need the field `authororder` in InfoObject? It is currently not set for publications.

**Comment by phish108** [comment]
Yes we do. Currently, the authors are ordered by their database position aka random. This does not reflect the actual order for the item. 

The idea is that the importer sets the authors AND the correct order. This requires a separate object that keeps the intrinsic order of the authors for the infoobject.
